rustc requirement is 1.45 not 1.44

```
error[E0658]: procedural macros cannot be expanded to statements
  --> /usr/local/cargo/registry/src/github.com-1ecc6299db9ec823/fixed-macro-impl-1.0.0/src/dispatch.rs:8:9
   |
8  | /         paste! {
9  | |             if ($int_bits, $frac_bits, $signed) == ($i, $f, true) {
10 | |                 return fixed::types::[<I $i F $f>]::from_str($s)
11 | |                     .map(|x| x.to_bits()).map(Literal::[<i $w _unsuffixed>])
...  |
15 | |             }
16 | |         }
   | |_________^
...
27 |       fixed_to_literal!(int_bits, frac_bits, signed, s, 8, 8, 0);
   |       ----------------------------------------------------------- in this macro invocation
   |
   = note: see issue #54727 <https://github.com/rust-lang/rust/issues/54727> for more information
   = note: this error originates in a macro (in Nightly builds, run with -Z macro-backtrace for more info)
```

--- a/.github/workflows/ci.yml
+++ b/.github/workflows/ci.yml
@@ -20,6 +20,14 @@ jobs:
           toolchain: ${{matrix.rust}}
       - run: cargo test
 
+  msrv:
+    name: Rust 1.45.0
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: dtolnay/rust-toolchain@1.45.0
+      - run: cargo check --tests
+
   clippy:
     name: Clippy
     runs-on: ubuntu-latest

--- a/README.md
+++ b/README.md
@@ -16,7 +16,7 @@ easily creating fixed-point constants for all of the fixed-point types provided 
 fixed-macro = "1.0"
 ```
 
-*Compiler support: same as [`fixed`][fixed] crate, rustc 1.44+ as of the time of writing.*
+*Compiler support: rustc 1.45+.*
 
 [fixed]: https://docs.rs/fixed
 [fixed-types]: https://docs.rs/fixed/latest/fixed/types/index.html

--- a/impl/src/dispatch.rs
+++ b/impl/src/dispatch.rs
@@ -5,15 +5,13 @@ use proc_macro2::Literal;
 
 macro_rules! fixed_to_literal {
     ($int_bits:expr, $frac_bits:expr, $signed:expr, $s:expr, $w:expr, $i:expr, $f:expr) => {
-        paste! {
             if ($int_bits, $frac_bits, $signed) == ($i, $f, true) {
-                return fixed::types::[<I $i F $f>]::from_str($s)
-                    .map(|x| x.to_bits()).map(Literal::[<i $w _unsuffixed>])
+                return paste![fixed::types::[<I $i F $f>]::from_str]($s)
+                    .map(|x| x.to_bits()).map(paste![Literal::[<i $w _unsuffixed>]])
             } else if ($int_bits, $frac_bits, $signed) == ($i, $f, false) {
-                return fixed::types::[<U $i F $f>]::from_str($s)
-                    .map(|x| x.to_bits()).map(Literal::[<u $w _unsuffixed>])
+                return paste![fixed::types::[<U $i F $f>]::from_str]($s)
+                    .map(|x| x.to_bits()).map(paste![Literal::[<u $w _unsuffixed>]])
             }
-        }
     };
 }
 

--- a/src/lib.rs
+++ b/src/lib.rs
@@ -7,7 +7,7 @@
 //! fixed-macro = "1.0"
 //! ```
 //!
-//! *Compiler support: same as [`fixed`][fixed] crate, rustc 1.44+ as of the time of writing.*
+//! *Compiler support: rustc 1.45+.*
 //!
 //! [fixed]: https://docs.rs/fixed
 //! [fixed-types]: https://docs.rs/fixed/latest/fixed/types/index.html

